RxSwift/Schedulers/SerialDispatchQueueScheduler.swift

```diff
@@ -27,41 +27,46 @@ In case some customization need to be made on it before usage,
 internal serial queue can be customized using `serialQueueConfiguration`
 callback.
 */
-public class SerialDispatchQueueScheduler : SchedulerType {
+public class SerialDispatchQueueScheduler: SchedulerType {
     public typealias TimeInterval = Foundation.TimeInterval
     public typealias Time = Date
-    
-    /// - returns: Current time.
```

Comment:
Please revert this change. `returns: Something` is the correct syntax Xcode detects.